code/game/objects/items/corgiclothes.dm

```diff
@@ -0,0 +1,20 @@
+//clothes specifically for Corgis. I had no idea where else to put this so why not make it its own big special thing
+
+/obj/item/corgiclothes
```

Comment:
Because if you give them a regular hardsuit and it magically morphs into a corgi hardsuit, that's just crazy! We need more REALISM in my spess simulator, especially where dog-spacesuits are concerned!